psi4/src/psi4/libfock/cubature.cc

```diff
@@ -2141,14 +2141,14 @@ void RadialGridMgr::GolombWelsch(int n, double a[], double b[], double q[])
 
 void RadialGridMgr::getLegendreRoots(int n, double r[], double w[])
 {
-    double a[n], bhack[n+1];
-    double *const b = &bhack[1]; // Note that b[n-1] is unused.
+    std::vector<double> a(n), bhack(n+1);
+    double *const b = &(bhack.data()[1]); // Note that b[n-1] is unused.
```

Comment:
Isn't `bhack.data()` enough here?

Reply:
`b` has to point to the 2nd element, while `bhack.data()` return a pointer to the 1st element.

Reply:
True that :+1: